Applications/TransferLearning/CIFAR_Classification/jni/main_func.cpp

```diff
@@ -264,61 +245,59 @@ int getBatch_val(float **outVec, float **outLabel, bool *last,
  * @param[in]  arg 2 : resource path
  */
 int main(int argc, char *argv[]) {
-  if (argc < 2) {
-    std::cout << "./nntrainer_classification_func Config.ini\n";
+  if (argc < 3) {
+    std::cout << "./nntrainer_classification Config.ini resources\n";
```

Comment:
Also, how about `resources/` to emphasize trailing slash

Reply:
I will intentionally add a \ at the end of the data path.